### DISCUSSION TITLE
Unleash edge helm chart version

### DISCUSSION THREAD

**Discussion by pallimalilsuhail** [Unleash/unleash#8325] [General]
Hello, we are using the [ helm chart](https://github.com/Unleash/helm-charts/blob/main/charts/unleash-edge/Chart.yaml) for our custom deployment of Unleash Edge. I noticed that the chart uses version `v19.2`, even though the latest release of Unleash Edge is [v19.4.1](https://github.com/Unleash/unleash-edge/releases/tag/unleash-edge-v19.4.1). Is there a specific reason for this? Is `v19.2` a recommended version, or is there another explanation?

**Comment by chriswk** [comment]
Just that we haven't updated the Helm chart. You can bump the version manually by overriding it in your values setup

**Comment by chriswk** [comment]
There's an update coming to Edge later this week, so I'll add to our TODO list for the release that we need to remember to bump the helm chart as well.

**Reply by pallimalilsuhail** [reply]
Thanks @chriswk 👍🏽
